MultiTarget/UseTargetFrameworks.ps1

```diff
@@ -23,15 +23,15 @@
 #>
 Param (
     [Parameter(HelpMessage = "The target frameworks to enable.")]
-    [ValidateSet('all', 'all-uwp', 'wasm', 'uwp', 'winappsdk', 'wpf', 'gtk', 'macos', 'ios', 'droid', 'netstandard')]
+    [ValidateSet('all', 'all-uwp', 'wasm', 'uwp', 'wasdk', 'wpf', 'linuxgtk', 'macos', 'ios', 'android', 'netstandard')]
```

Comment:
We should check we don't reference these in any other scripts/build.yml files that need to be updated too, eh?

Or track very explicitly that we'll need to update them in Windows/Labs-Windows.

Reply:
Did a quick skim through our codebase for usages of these, think we got them all.

`build.yml` is using 'all' and 'all-uwp', so we're safe there.